PostgresDockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:9.6.24
```

Comment:
Postgres 9.6 is not supported since 2021 November 11 ([ref](https://www.postgresql.org/support/versioning/)).

Maybe we should take this opportunity and bump the db version as well.

Reply:
Good idea I made the bump. It seems to work fine :+1: